projects/storefrontstyles/scss/components/checkout/order-confirmation/_order-confirmation-overview.scss

```diff
@@ -1,59 +1,7 @@
 @import '../../../theme';
 
 %cx-order-confirmation-overview {
-  .cx-order-review-summary {
-    background-color: var(--cx-color-background);
-    border-width: 0 0 1px 0;
-    border-style: solid;
-    border-color: var(--cx-color-light);
-
-    @include media-breakpoint-down(sm) {
-      background-color: var(--cx-color-inverse);
-    }
-
-    .container {
-      display: flex;
-      flex-wrap: wrap;
-      padding: 15px 0;
-
-      @include media-breakpoint-down(md) {
-        max-width: 100%;
-        min-width: 100%;
-        padding: 0 1.25rem;
-      }
-      @include media-breakpoint-down(sm) {
-        flex-direction: column;
-        padding: 1.25rem;
-      }
-
-      .summary-card {
-        flex: 1;
-        padding: 0 15px;
-
-        &:not(:last-of-type) {
-          @include media-breakpoint-up(lg) {
-            border-inline-end: 1px solid var(--cx-color-text);
-          }
-        }
-
-        @include media-breakpoint-down(md) {
-          flex: 0 0 33%;
-        }
-
-        @include media-breakpoint-down(sm) {
-          flex: 1;
-          background-color: var(--cx-color-inverse);
-          border-width: 1px;
-          border-style: solid;
-          border-color: var(--cx-color-light);
-          margin: 0.625rem 0;
-        }
-
-        .cx-card-title {
-          @include type('4');
-          font-weight: bold;
-        }
-      }
-    }
-  }
+  /**
```

Comment:
Why is this?

Reply:
I was wondering if we should just leave it there for future usage or remove it entirely.

What do you think I should do

Reply:
removed file and selectors from index